src/__mocks__/UIManager.js

```diff
@@ -90,25 +93,22 @@ var RCTUIManager = {
   setJSResponder: jest.fn(),
   setChildren: jest.fn(function setChildren(parentTag, reactTags) {
     autoCreateRoot(parentTag);
+    /* TODO (spicyj) Re-enable this check once it won't cause test failures
     // Native doesn't actually check this but it seems like a good idea
     invariant(
       views.get(parentTag).children.length === 0,
       'Calling .setChildren on nonempty view %s',
       parentTag,
     );
+    */
     // This logic ported from iOS (RCTUIManager.m)
     reactTags.forEach((tag, i) => {
       insertSubviewAtIndex(parentTag, tag, i);
     });
   }),
-  manageChildren: jest.fn(function manageChildren(
-    parentTag,
-    moveFromIndices = [],
-    moveToIndices = [],
-    addChildReactTags = [],
-    addAtIndices = [],
-    removeAtIndices = [],
-  ) {
+  manageChildren: jest.fn(function manageChildren(parentTag, moveFromIndices = [
+  ], moveToIndices = [], addChildReactTags = [], addAtIndices = [
+  ], removeAtIndices = []) {
```

Comment:
I fixed this yesterday. I'll do a release this morning with the fix. Sorry about this!

It's going to look like this:
```js
  manageChildren: jest.fn(function manageChildren(
    parentTag,
    moveFromIndices = [],
    moveToIndices = [],
    addChildReactTags = [],
    addAtIndices = [],
    removeAtIndices = []
  ) {
    autoCreateRoot(parentTag);
```